doc/INSTALL.md

```diff
@@ -194,7 +194,7 @@ You can build Mountpoint for Amazon S3 from source. However, binaries built in t
 4. Build Mountpoint for Amazon S3:
    ```
    cd mountpoint-s3
-   cargo build --release
+   cargo +nightly build --release
```

Comment:
There should be no need to use the nightly toolchain. We build mountpoint with stable (currently rustc 1.78.0).

Reply:
Are you seeing any issues without it?

Reply:
Ahh maybe I was using an older stable which only had a feature in nightly. Perhaps this doc could then have a note/hint about not using nightly and upgrading to latest stable.

Reply:
Better yet, the project could use `rust-toolchain.toml`